src/mapping/nav2_map_server/src/map_reps/occ_grid_server.cpp

```diff
@@ -314,7 +315,11 @@ void OccGridServer::OccMapCallback(
 {
   (void)request_header;
   (void)req;
+//  occ_resp_.map = map_msg_;
```

Comment:
I suppose we could get rid of the map_msg_ and only work with occ_resp_.map throughout the file.